OrganizationSearch/FindOrganization.js

```diff
@@ -7,18 +7,26 @@ import {
   PLUGIN_RESULT_COUNT_INCREMENT,
 } from '@folio/stripes-acq-components';
 
-import { OrganizationsListFilter } from './OrganizationsListFilter';
+import {
+  DEFAULT_DONOR_FILTERS,
+  DEFAULT_VISIBLE_COLUMNS,
+  DONOR_COLUMNS,
+} from './constants';
 import { useOrganizations } from './hooks';
-
-import { searchableIndexes } from './OrganizationsSearchConfig';
+import { OrganizationsListFilter } from './OrganizationsListFilter';
+import {
+  donorsSearchableIndexes,
+  searchableIndexes as organizationSearchableIndexes,
+} from './OrganizationsSearchConfig';
 
 const INIT_PAGINATION = { limit: PLUGIN_RESULT_COUNT_INCREMENT, offset: 0 };
 
 const idPrefix = 'ui-plugin-find-organization-';
-const modalLabel = <FormattedMessage id="ui-plugin-find-organization.modal.label" />;
-const resultsPaneTitle = <FormattedMessage id="ui-plugin-find-organization.meta.pluginTitle" />;
+const defaultModalLabel = <FormattedMessage id="ui-plugin-find-organization.modal.label" />;
+const donorsModalLabel = <FormattedMessage id="ui-plugin-find-organization.donors.modal.label" />;
```

Comment:
again, please apply configuration for plugin, it shouldn't know about donor or not donor